net-tools/netwatch/src/netmon/windows.rs

```diff
@@ -100,16 +100,19 @@ impl CallbackHandler {
         let mut handle = Handle::default();
         let cb = Arc::new(cb);
         unsafe {
-            windows::Win32::NetworkManagement::IpHelper::NotifyUnicastIpAddressChange(
+            let r = windows::Win32::NetworkManagement::IpHelper::NotifyUnicastIpAddressChange(
```

Comment:
This is stupid but we can no longer just `?` the error we have to parse it manually.

Reply:
wähh